lib/migration-steps/validation.js

```diff
@@ -14,6 +14,7 @@ const contentTypePropsValidations = {
 
 const fieldPropsValidations = {
   id: Joi.string().required(),
+  newId: Joi.string(),
```

Comment:
I think this needs to be required as well, as we don't use the whole object as a schema, but pick by property name and use the value as the schema.

So basically `Joi.validate(value, Joi.string())` is called, which would allow for empty content, am I right?

Reply:
It wouldn't allow for an empty string but it would allow no value at all. Thanks for the catch, updating..